lstchain/reco/dl1_to_dl2.py

```diff
@@ -144,12 +140,8 @@ def train_disp_vector(train, features,
     The trained model
     """
     if config_file is not None:
-        try:
-            config = read_configuration_file(config_file)
-            regression_args = config['random_forest_regressor_args']
-            print("Configuration loaded from {}".format(config_file))
-        except:
-            print("Configuration could not be loaded from file. Standard configuration applies")
+        config = read_configuration_file(config_file)
```

Comment:
Same comment for the next ones

Reply:
If it is the same comment, then could we discuss it only in one place and accodingly resolve this convversation?

Reply:
yes